charts/mokta/values.yaml

```diff
@@ -19,6 +19,7 @@ ingress:
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/ssl-redirect: '443'
     alb.ingress.kubernetes.io/tags: Environment=review,Product=mokta
+    alb.ingress.kubernetes.io/healthcheck-path: /status
```

Comment:
I don't think Mokta has a /status endpoint @mrdaniellewis ?

Reply:
It's got a version endpoint https://mokta.devops.citizensadvice.org.uk/version